install.sh

```diff
@@ -0,0 +1,7 @@
+#/bin/bash
+
+git clone https://github.com/PurpLabs/PogBot.git
+cd PogBot
+pip install -r requirements.txt
```

Comment:
this project uses the poetry package manager— so the script should either export the requirements to requirements.txt or just create a new environment with poetry (what i recommend)